deps.edn

```diff
@@ -17,19 +17,10 @@
   org.eclipse.elk/org.eclipse.elk.alg.layered {:mvn/version "0.10.0"}}
 
  :aliases
- ;; Build the site with `clojure -M:m`
- {:m     {:main-opts ["-m" "civitas.main"]}
-  ;; Run Clay in watch mode with `clojure -J-Dclojure.main.report=stderr -M:clay`
```

Comment:
If you still want the stderr reporting, you could add `:jvm-opts ["-Dclojure.main.report=stderr"]` to the `:clay` alias.